src/jcstress/java/reactor/pool/SimpleDequePoolStressTest.java

```diff
@@ -24,15 +25,73 @@
 import org.openjdk.jcstress.annotations.JCStressTest;
 import org.openjdk.jcstress.annotations.Outcome;
 import org.openjdk.jcstress.annotations.State;
+import org.openjdk.jcstress.infra.results.IIII_Result;
 import org.openjdk.jcstress.infra.results.II_Result;
 
 import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
 
-import static org.openjdk.jcstress.annotations.Expect.ACCEPTABLE;
-import static org.openjdk.jcstress.annotations.Expect.FORBIDDEN;
+import static org.openjdk.jcstress.annotations.Expect.*;
 
 public class SimpleDequePoolStressTest {
 
+	@JCStressTest
+	@Outcome(id = "1001, 2, 0, 0", expect = ACCEPTABLE,  desc = "evicted, acquired second resource")
```

Comment:
4 state variables make it _really_ hard to reason about this test. Given that 1st and 3rd are always "1001" and "0", can't we get rid of them?

Reply:
I don't think so, no. Both these validate that the pool correctly released the resource and that the accounting is correct. Maybe put them as 3rd and 4th variable since slightly less interesting than the second one (acquired)